static/katex.less

```diff
@@ -22,6 +22,12 @@
     // Protect elements inside .katex from inheriting text-indent.
     text-indent: 0;
 
+    // Prevent a rendering bug that misplaces \vec in Chrome.
+    text-rendering: auto;
+  
+    // Prevent background resetting in Windows's high-contrast mode.
+    -ms-high-contrast-adjust: none !important;
```

Comment:
This was replaced by `* { -ms-high-contrast-adjust: none !important; }` a couple of lines below in https://github.com/Khan/KaTeX/pull/733 and should be removed.